opExit.cpp

```diff
@@ -0,0 +1,42 @@
+#include"operations/opExit.h"
+
+#include "..\controller.h"
+#include "../Shapes/Shape.h"
+
+#include "..\GUI\GUI.h"
+
+#include "Shapes/Graph.h"
+#include "operations/operation.h"
+
+
+opExit::opExit(controller* pCont) :operation(pCont)
+{}
+opExit::~opExit()
+{}
+
+//Execute the operation
+void opExit::Execute() {
+	
+	//Get a Pointer to the Input / Output Interfaces
+	GUI* pUI = pControl->GetUI();
```

Comment:
You have to check if the user drew anything and then check if it is saved or not. Because if he didn't draw anything then it doesn't make sense to tell him there are changes unsaved and then it will save an empty file that will be an issue in the Load function.